### PR TITLE
[Update] Code updates according to pagespeed advice

### PR DESCRIPTION
**Pull request by nickhsine**
- defer script tag to avoid blocking
- load typekit asynchronously
- not to lazy load leading image

### CODE REVIEW

**Comment by nickhsine** [comment]
@garfieldduck please help to review it

File: src/components/article/LeadingImage.js [hunk 1]
```diff
@@ -34,12 +50,17 @@ class LeadingImage extends React.Component {
         break
     }
 
+    const defaultImgUrl = replaceStorageUrlPrefix(_.get(image, 'tiny.url'))
+
+    const imgJsx = this.state.isMount ? (
+      <img src={defaultImgUrl} alt={description} srcSet={getImageSrcSet(image)} />
+    ) : (
+      <img src={defaultImgUrl} alt={description} />
```
Comment (garfieldduck) [inline src/components/article/LeadingImage.js]: do you want to add a blurry class/effect for the first loaded image?

Comment (nickhsine) [inline src/components/article/LeadingImage.js]: The leading image will be a tiny image at first.

Comment (garfieldduck) [inline src/components/article/LeadingImage.js]: Yes. I mean maybe you could add a blurry image class to the first loaded tiny image.

**Pull request merged**